### PR TITLE
docs(developers/server-security): fix typo in server-security.md

### PR DESCRIPTION
**Pull request by mardev-xyz**

### CODE REVIEW

**Review by colistro123 — CHANGES_REQUESTED** [review]
Hey there, suggested a change.

**Comment by PsychoShock** [comment]
Ah oops, me and my bad English grammar 🙈

**Comment by mardev-xyz** [comment]
> Hey there, suggested a change.

Added the suggestion :)

File: content/docs/developers/server-security.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ Connection to CNL timed out.
 
 **Note:** some error *might occur* but it's unlikely.
 
-User getting kicked doesn't mean is automatically Global Ban. However, it provides an indication of player reliability, which can be useful for assessing trustworthiness.
+Kicking a user does not necessarily lead to a Global Ban. However, it provides an indication of player reliability, which can be useful for assessing trustworthiness.
```
Comment (AvarianKnight) [inline content/docs/developers/server-security.md:211]:
```suggestion
The player getting kicked does not necessarily mean a malicious player.
```
We're better off not leading people to believe that these are 100% accurate currently.

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Comment by mardev-xyz** [comment]
Added requested changes, once again!

**Comment by mardev-xyz** [comment]
@AvarianKnight Your change has been added aswell!

**Comment by AvarianKnight** [comment]
This will be superseded by https://github.com/citizenfx/fivem-docs/pull/501

File: content/docs/developers/server-security.md [hunk 2]
```diff
@@ -208,7 +208,7 @@ Connection to CNL timed out.
 
 **Note:** some error *might occur* but it's unlikely.
 
-User getting kicked doesn't mean is automatically Global Ban. However, it provides an indication of player reliability, which can be useful for assessing trustworthiness.
+The player getting kicked does not necessarily mean a malicious player.
```
Comment (colistro123) [inline content/docs/developers/server-security.md:211]: I will implement this change in the other PR and will credit this PR for it 😀

**Pull request closed without merging**